src/ui/public/vislib/lib/axis/axis.js

```diff
@@ -266,7 +163,7 @@ export default function AxisFactory(Private) {
 
     validate() {
       if (this.axisConfig.isLogScale() && this.axisConfig.isPercentage()) {
-        throw new errors.VislibError(`Can't mix percentage mode with log scale.`);
```

Comment:
yeah i know ... for some reason i couldn't push as eslint insisted i put single quotes here :\